jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpTester.java

```diff
@@ -504,6 +510,14 @@ public ByteBuffer generate()
             }
         }
 
+        public byte[] generateBytes()
+        {
+            ByteBuffer byteBuffer = generate();
+            byte[] bytes = new byte[byteBuffer.remaining()];
+            byteBuffer.get(bytes);
+            return bytes;
```

Comment:
Don't think this method is needed.

This can be shortened to ...

`return BufferUtil.toArray(generate());`

Which is how it's used in test cases.

Either as a String ...

``` java
socket.write(BufferUtil.toString(request.generate()));
```

or as a byte array ...

``` java
socket.write(BufferUtil.toArray(request.generate()));
```

Reply:
Good point, removed.